uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -177,8 +177,7 @@ define([
           .then(function (balanceChange) {
             if (balanceChange) {
               self.balance.change = balanceChange;
-              // the change is negative so adding it will actually subtract it
-              self.balance.closing = self.balance.opening + self.balance.change.amount;
+              self.calculateClosingBalanceChange();
```

Comment:
https://github.com/civicrm/civihr/blob/bb675fc47a017056d17d1683b8ad6144af653181/uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/toil-request-ctrl.js#L34-L34

`toil-request-ctrl`overrides the `calculateBalanceChange` function, so `calculateClosingBalanceChange` needs to be used there as well.